### PR TITLE
Word Break

### PR DESCRIPTION
**Pull request by Meghag5**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

(yes)

### Description

(This problem is solved using backtracking it divides the string into words and pass all the valid strings which can be formed with those words)

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests
#2295 
(Write your answer here.)

### CODE REVIEW

File: C-Plus-Plus/README.md [hunk 1]
```diff
@@ -218,6 +218,13 @@ _add list here_
 
 - [Convex Hull (Jarvis' Algorithm)](computational_geometry/Convex_Hull_Jarvis_Algorithm.cpp)
 
+<<<<<<< HEAD
+## Backtracking
+-[Word Break Problem](backtracking/Word_break.cpp)
+=======
+
+>>>>>>> f3f8e3ecff01def04109ea96784a01ca0ec9414f
```
Comment (HarshCasper) [inline C-Plus-Plus/README.md:226]: Why are the commit hashes in the README now 🤷‍♂️

**Review by HarshCasper — CHANGES_REQUESTED** [review]
You have committed binary files into the PR. Remove them.

Also we won't review PRs having more than one Code File change. Either remove the other code file or break this PR into two

**Comment by Meghag5** [comment]
> You have committed binary files into the PR. Remove them.
> 
> Also we won't review PRs having more than one Code File change. Either remove the other code file or break this PR into two

should I send it again on new branch?

**Comment by Meghag5** [comment]
> > You have committed binary files into the PR. Remove them.
> > Also we won't review PRs having more than one Code File change. Either remove the other code file or break this PR into two
> 
> should I send it again on new branch?

pls help me as I am a beginner and i face problem using git how should I remove extra files it is showing

**Comment by HarshCasper** [comment]
@Meghag5 

Drop me a message on Discord. Don't panic and just let us know on the apt channels.

**Pull request closed without merging**